pip install PIL (external dependencies) fails on linux

choosing the method of installing python deps with `pip install -r external-deps` ..as provided in the Readme, my build fails on linux. 

The reason is obvious (freetype vs freetype2), thing is how/who should solve it: 
a) my distro (Arch) should install in freetype/ 
b) PIL's pip package fault (for not thinking of this possibility)
c) we could help pip by setting include_dirs somehow (that's what appeared on a quick google try)
d) drop PIL from default install. - when I remove PIL from the external-reqs, pip install -r and following ./cleanbuild both run fine. I remember PIL was there, then dropped, and then reintroduced again..what part of code uses it? 

building '_imagingft' extension

gcc -pthread -fno-strict-aliasing -march=x86-64 -mtune=generic -O2 -pipe -fstack-protector --param=ssp-buffer-size=4 -DNDEBUG -march=x86-64 -mtune=generic -O2 -pipe -fstack-protector --param=ssp-buffer-size=4 -fPIC -I/usr/include/freetype2 -IlibImaging -I/home/marek/devel/nupic/py27/include -I/usr/local/include -I/usr/include -I/usr/include/python2.7 -c _imagingft.c -o build/temp.linux-x86_64-2.7/_imagingft.o

_imagingft.c:73:31: fatal error: freetype/fterrors.h: No such file or directory

 #include <freetype/fterrors.h>

```
                           ^
```

compilation terminated.

error: command 'gcc' failed with exit status 1

---

Cleaning up...
Command /home/marek/devel/nupic/py27/bin/python2.7 -c "import setuptools, tokenize;**file**='/home/marek/devel/nupic/py27/build/PIL/setup.py';exec(compile(getattr(tokenize, 'open', open)(**file**).read().replace('\r\n', '\n'), **file**, 'exec'))" install --record /tmp/pip-djckB0-record/install-record.txt --single-version-externally-managed --compile --install-headers /home/marek/devel/nupic/py27/include/site/python2.7 failed with error code 1 in /home/marek/devel/nupic/py27/build/PIL
Traceback (most recent call last):
  File "/home/marek/devel/nupic/py27/bin/pip", line 11, in <module>
    sys.exit(main())
  File "/home/marek/devel/nupic/py27/lib/python2.7/site-packages/pip/**init**.py", line 185, in main
    return command.main(cmd_args)
  File "/home/marek/devel/nupic/py27/lib/python2.7/site-packages/pip/basecommand.py", line 161, in main
    text = '\n'.join(complete_log)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 72: ordinal not in range(128)
(py27)[marek@beruska nupic]$ find /usr/include/ -type f -name fterrors.h
/usr/include/freetype2/fterrors.h

--- a/external/common/requirements.txt
+++ b/external/common/requirements.txt
@@ -2,7 +2,7 @@
 asteval==0.9.1
 mock==1.0.1
 ordereddict==1.1
-PIL==1.1.7
+pillow==2.3.0
 psutil==1.0.1
 pylint==0.28.0
 pytest==2.4.2

